docs/usecases/REMoDNaV.rst

```diff
@@ -23,20 +23,20 @@ and results and paper can be generated and recomputed everywhere on demand.
 The Problem Space
 ^^^^^^^^^^^^^^^^^
 Over the past year, Steve worked on the implementation of an algorithm as a software package.
-For testing purposes, he used one of his own data sets, and later also included a publicly shared
-data set. After completion, he continued to work on validation analyses to
+For testing purposes, he used one of his own data collection, and later also included a publicly shared
```

Comment:
Just a small fix. 

"data collection" should be "data collections"

Reply:
thx, @loj!